components/logins/src/db.rs

```diff
@@ -419,6 +419,82 @@ impl LoginDb {
         Ok(login)
     }
 
+    pub fn import_multiple(&self, logins: &[Login]) -> Result<u64> {
```

Comment:
We expect the DB to be empty now, right? And for this not to be called except for non-sync users?

Is it worth failing if these things are untrue?